xml/art-obs-beginners-guide.xml

```diff
@@ -56,45 +56,44 @@ https://www.suse.com/communities/blog/suse-studio-integration/
   <sect2 xml:id="sec.obsbg.concept.pkgspec">
    <title>Build Recipe</title>
    <para>
-    To create a package in &obsa;, you need a <firstterm>build recipe</firstterm>
-    which contains the following information:
+    To create a package in &obsa;, you need to provide the following
+    information:
    </para>
    <itemizedlist>
     <listitem>
      <formalpara>
       <title>Metadata</title>
-      <para>The package name and the description are mandatory. Other data
-       such as the version, the license, the upstream URL
-       is optional.</para>
+      <para>This is the package name, its description, possibly
+      license information...</para>
      </formalpara>
     </listitem>
     <listitem>
      <formalpara>
-      <title>Requirements</title>
+      <title>Information on how to build</title>
       <para>
-       Packages depend on other packages to function properly.
-       There are two types of requirements:
-       <emphasis>build requirements</emphasis> and
-       <emphasis>installation requirements</emphasis>.
+        You need to indicate how to produce the files that will be
+        part of the package: where to get the sources, and which
+        command(s) to run, e.g. <command>make</command>.
+      </para>
+      <para>
+        The build commands generally requires that some other packages
+        are installed beforehand; for example, to build a C++ program,
+        you need a C++ compiler.  You need to indicate this list of
+        packages: the <emphasis>build requirements</emphasis>.
```

Comment:
IMHO this is way to broad for our documentation here. We do not want to document package formats in general here, just concepts and OBS specifics.

Reply:
I think @tomschr wrote this from the perspective of OBS allowing to create many types of packages/images, but RPM still being the most important of those. As such, it does make sense to me. I also would not overburden the Beginner's Guide with too many theoretical concerns that do not affect RPM.

(I have to say that on the whole, this PR does not make much sense to me in its current form -- the wording updates it makes generally seem to make it less specific/useful.)

Reply:
Thanks a lot for your review, it is much appreciated.
On my side, I do deb packages and I'm interested in OBS to perform this task, so I'm actually trying to learn from its documentation; however certain things do not map correctly for deb packages and that's the main reason I'm proposing changes.  I'm trying to make it more universally correct while not complicating the reading.  My goal is that the next person in my case will have a less hard time.
I'm happy to take any advice to make it better.

Reply:
Maybe it would make sense to create two Beginners Guides: one for RPM base (openSUSE) and one for DEB (Debian)?
It also depends probably on how much text we could share between the two.

Reply:
> Maybe it would make sense to create two Beginners Guides: one for RPM base (openSUSE) and one for DEB (Debian)?
> It also depends probably on how much text we could share between the two.

# Packaging

For openSUSE based RPMs, we usually push people to read the [english openSUSE wiki](https://en.opensuse.org/Portal:Packaging). This is our reference documentation. Especially as it allows anyone to quickly adjust the content, once new features (including package macros) end up in the openSUSE distribution. With this, I would recommend to exclude the openSUSE packaging part completely and instead refer to the wiki.

About [Debian](https://en.opensuse.org/openSUSE:Build_Service_Debian_builds) / [Fedora](https://en.opensuse.org/Fedora_packaging) - or any other packaging: IMHO we should only refer to the specific pages in the openSUSE wiki. Things are really changing quickly - and people usually tend to not send PRs to Github, but edit the wiki instead. 

# Administration

A big difference are IMHO the configuration settings for the different distributions. 

These settings have to be done by the administrators of the Build Service. Here the wiki currently lacks this documentation, which is no surprise, as [OBS](https://build.opensuse.org) usually/magically (thanks to Adrian) provides them. But this is also what makes the live of any OBS admin hard: he has to figure out "how Adrian did this"? :-)